### DISCUSSION TITLE
请求退款mail：1427780725@qq.com

### DISCUSSION THREAD

**Discussion by ypx0000** [getlantern/lantern-forum-cn#930] [General]
用不了，我需要退款
![66177be43885d54b04e8bbc059b5cb8](https://github.com/user-attachments/assets/3b2270e8-2cdb-4fbb-a48b-9093b079e857)

**Comment by ypx0000** [comment]
麻烦帮我退款谢谢

**Comment by ypx0000** [comment]
官方人员看到能帮我退款吗

**Comment by MaxMKMax** [comment]
@ypx0000 您好. 请问您是遇到什么问题无法使用吗? 我们能不能帮您解决问题? 如果您愿意让我们解决的话就不需要退款了. 希望您先别退款.
如果您坚持要退款, 我帮您申请退款.

**Comment by ypx0000** [comment]
***@***.***这就是我的账号邮箱




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "getlantern/lantern-forum-cn"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年10月10日(星期四) 凌晨2:55
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [getlantern/lantern-forum-cn] ***@***.*** (Discussion #930)





 
@ypx0000 您好. 请问您是遇到什么问题无法使用吗? 我们能不能帮您解决问题? 如果您愿意让我们解决的话就不需要退款了. 希望您先别退款.
 如果您坚持要退款那请您提供我账号邮箱. 我帮您申请退款.

**Comment by MaxMKMax** [comment]
@ypx0000 您好. 已经帮您提交问题上去了

**Comment by MaxMKMax** [comment]
@ypx0000 
您好. 这是您的激活码:
GXQRB-3T99R-7FX9Q-DGQ33-9D4RK
